src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../styles/variables.css';
+
+.button {
+  --button-color: var(--color-base-white);
+  --button-color-background: var(--color-brand-primary-base);
+  --button-color-background-hover: var(--color-brand-primary-dark);
+  --button-color-background-disabled: var(--color-brand-primary-light);
+
+  // small
+  --button-size-sm-border-radius: 4px; // var(--size-border-radius-sm);
+  --button-size-sm-font-size: var(--size-font-sm);
+  --button-size-sm-padding: var(--size-spacing-xs) var(--size-spacing-sm);
+
+  // medium (default)
+  --button-size-md-border-radius: 8px; // var(--size-border-radius-md);
+  --button-size-md-font-size: var(--size-font-md);
+  --button-size-md-padding: var(--size-spacing-sm) var(--size-spacing-md);
+
+  // large
+  --button-size-lg-border-radius: 8px; // var(--size-border-radius-md);
+  --button-size-lg-font-size: var(--size-font-lg);
+  --button-size-lg-padding: var(--size-spacing-md) var(--size-spacing-lg);
```

Comment:
@juanfabrega we talked about this offline, but can't remember what we decided. do you think we should move these to `/styles/variables.css`?

Reply:
@nathanyoung I am currently working on this re-structure.